content/chainguard/chainguard-images/working-with-images/unique-tags/index.md

```diff
@@ -43,17 +43,17 @@ Additionally, the Unique Tags feature is integrated with Chainguard's [Tag Histo
 
 After signing into the Chainguard Console, click on **Organization images** in the left-hand navigation. This will take you to your organization's Images directory where you'll be presented with a list of all the Chainguard Production Images your organization has access to.
 
-To the right of the search box and **Category** drop-down menu is a toggle button labeled **Include unique tags**. 
+To the right of the search box and **Category** drop-down menu there's a filter button labeled **Visible tags**. Click on that button, and you'll see a drop-down menu with two options: **Epoch tags** and **Unique tags**. Toggle **Unique tags** to see the Unique Tags available for your organization's Images.
 
-![Screenshot of the Organization images page, showing four Images in a table. The "Include unique tags" button is highlighted with a yellow box.](unique-tags-1.png)
+![Screenshot of the Organization images page, showing Images in a table. The "Visible Tags" select box is highlighted with a yellow box, showing the option "Unique tags" checked.](unique-tags-01.png)
 
 By toggling this button on, each individual Image's details page will show the Unique Tags available for it.
 
-To illustrate, toggle this button on and then click on any Production Image listed in your organization's directory. This example shows a partial list of available versions of the `busybox` Image from the **Versions** tab on its details page.
+To illustrate, toggle this button on and then click on any Production Image listed in your organization's directory. The "Version" column will now show the Unique Tags available for that Image. These tags include a timestamp in the format `YYYYMMDDHHMM`, and may include a prefix to help identify and parse the tag name programmatically.
 
-![Screenshot of a portion of the busybox Image's Versions tab. This screenshot shows the ten most recently built versions, some of which include unique tags such as "1-202312280512"](unique-tags-2.png)
+![This screenshot shows the ten most recently built versions of an image that has unique tags enabled.](unique-tags-02.png)
```

Comment:
It's strange, this image looks super blurry in the preview :thinking:

Reply:
ooof you're right! I am trying some optimizations now to see if I can make it look better.  the image itself is large, I think that I might need to resize down.